charts/images.yaml

```diff
@@ -28,6 +28,10 @@ images:
   sourceRepository: github.com/gardener/etcd-druid
   repository: eu.gcr.io/gardener-project/gardener/etcd-druid
   tag: "v0.15.3"
+- name: fluent-operator
+  sourceRepository: https://github.com/fluent/fluent-operator
+  repository: kubesphere/fluent-operator
```

Comment:
The [documentation](https://cloud.google.com/container-registry/docs/pulling-cached-images) about mirror.gcr.io contains the following statements
> Container Registry adds frequently requested images to the cache so they are available for future requests. It also periodically removes images that are no longer requested.


and 

> However, there is no guarantee that a particular image will remain cached for an extended period of time. Only obtain cached images on [mirror.gcr.io](http://mirror.gcr.io/) by [configuring the Docker daemon](https://cloud.google.com/container-registry/docs/pulling-cached-images#configure).


Based on this, I don't think using mirror.gcr.io in images.yaml is an option and this will soon start causing issues as the images will not be found in the registry. Even those that are deploying Gardener with image vector overwrite might not be able to generate the image vector overwrite.

Reply:
cc @breuerfelix since https://github.com/gardener/gardener/pull/7561/commits/2fd6965c6d3ef603c89b2af292470ca54a21e3da builds on the GCR mirror

Probably we should rather push the images to the regular gardener GCR then.

Reply:
Maybe we should also open a discussion about the image registry and if we want to trust the mirror or better copy needed dockerhub images to the gardener third party GCR. cc @timebertt (since we discussed this topic in slack)

Reply:
Thanks, everybody for bringing this up.
I opened https://github.com/gardener/ci-infra/issues/619 to tackle this and will perform the necessary steps to set this straight.

Reply:
https://github.com/gardener/ci-infra/issues/619 is ready and you can start using the prow job for copying images: https://github.com/gardener/ci-infra/tree/master/config/images.
The documentation is adapted accordingly in https://github.com/gardener/gardener/pull/7698.